invectio/cli.py

```diff
@@ -93,20 +96,28 @@ def cli(ctx=None, verbose: bool = False) -> None:
     "--without-builtin-imports/--with-builtin-imports",
     is_flag=True,
     show_default=True,
+    help="Do not report usage of Python's standard library.",
+)
+@click.option(
+    "--without-builtins/--with-builtins",
+    is_flag=True,
+    show_default=True,
     help="Do not report usage of Python's builtins.",
 )
 def whatuses(
     path: str,
     ignore_errors: bool = False,
     without_standard_imports: bool = False,
     without_builtin_imports: bool = False,
+    without_builtins: bool = False,
```

Comment:
Why is the default False here and True inside of the InvectioLibraryUsageVisitor?

Reply:
Nice catch! Unified this logic in 7217eeb. PTAL 👍🏻